javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,31 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (!studentName) {
+    alert('"please enter a Name"');
+  } else if (class07Students.includes(studentName)) {
+    alert(`Student "${studentName}" is already in the class`);
+  } else if (studentName === "Dronning" || class07Students.length < 6) {
+    return class07Students.push(studentName);
+  } else if (class07Students.length >= 6) {
+    alert('"Cannot add more students to class 07"');
+  } else {
+    return class07Students.push(studentName);
```

Comment:
again, no need to return actions here. You use `return` when you literally want the function to return a value (like the string before), or when you want the function to stop running (it is at the end it it will stop running once it reaches the last condition if it didn't exit before already)